packages/assets-controllers/src/AccountTrackerController.test.ts

```diff
@@ -50,31 +59,42 @@ describe('AccountTrackerController', () => {
 
   it('should get real balance', async () => {
     const address = '0xc38bf1ad06ef69f0c04e29dbeb4152b4175f0a8d';
+
+    mockedQuery.mockReturnValueOnce(Promise.resolve('0x10'));
+
     const controller = new AccountTrackerController(
       {
         onPreferencesStateChange: sinon.stub(),
         getIdentities: () => {
           return { [address]: {} as ContactEntry };
         },
+        getSelectedAddress: () => '0x0',
```

Comment:
Q: Should the selected address be one of the addresses in the getIdentities result array?
Given the way it's defined in [PreferencesController.ts](https://github.com/MetaMask/core/pull/1146/files#diff-40a12640df782cc92dd07efa2b2f120024eb7c5cf1d9431893406ef39132e272R211-R215), it seems to be necessarily on of the identities. Either an explicit one or the first one by default.